.config/matugen/templates/hyprlunavsc.json

```
@@ -0,0 +1,1934 @@
+    "editorSuggestWidget.highlightForeground": "{{colors.primary.default.hex}}",
+    "editorSuggestWidget.selectedBackground": "{{colors.surface_variant.default.hex}}",
+    "editorHoverWidget.foreground": "{{colors.on_surface.default.hex}}",
+    "editorHoverWidget.background": "{{colors.surface.default.hex}}",
+    "editorHoverWidget.border": "{{colors.outline.default.hex}}",
+    "editorMarkerNavigation.background": "{{colors.surface.default.hex}}",
+    "editorMarkerNavigationError.background": "{{colors.error.default.hex}}40",
+    "editorMarkerNavigationWarning.background": "{{colors.secondary.default.hex}}40",
+    "editorMarkerNavigationInfo.background": "{{colors.tertiary.default.hex}}40",
+    "peekView.border": "{{colors.primary.default.hex}}",
+    "peekViewEditor.background": "{{colors.surface.default.hex}}",
+    "peekViewEditor.matchHighlightBackground": "{{colors.tertiary.default.hex}}40",
```

Biome (2.1.2)

[error] The key peekViewEditor.matchHighlightBackground was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)